content/en/monitors/guide/custom_schedules.md

```diff
@@ -0,0 +1,104 @@
+
+For example, the following monitor checks every day at 8:00PM that a success event has been generated for each database instance as a result of the daily backup job. 
+
+{{< img src="monitors/guide/custom_schedules/custom_day.png" alt="Monitor configuration to check every day at 8pm that a success event has been generated for each database instance as a result of the daily backup job" style="width:100%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Week" %}}
+Select the days of the week as well as the time of the day you want the monitor to evaluate at.
+
+For example, the following monitor checks every week on Tuesday and Saturday at 6:00AM that marketing emails have been sent for each individual campaign. 
+
+{{< img src="monitors/guide/custom_schedules/custom_week.png" alt="Monitor configuration to check every week on Tuesday and Saturday at 6am that marketing emails have been sent for each individual campaign" style="width:100%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Month" %}}
+Select the day of the month as well as the time of the day you want the monitor to evaluate at.
+
+For example, the following monitor checks on the first day of each month whether the cron-job generating customer invoices has successfully run.
+
+{{< img src="monitors/guide/custom_schedules/custom_month.png" alt="Monitor configuration to check on the first day of each month whether the cron-job generating customer invoices has successfully run." style="width:100%;" >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## RRULES
+
+Recurrence rule (RRULE) is a property name from [iCalendar RFC][1], which is the standard for defining recurring events. Use the [official RRULE generator][2] as a tool to generate recurring rules. Leverage RRULEs to cover more advanced scheduling use-cases. 
+
+To write a custom RRULE for your monitor, click **Use RRULE**.
+
+**Notes**: 
+- Attributes specifying the duration in RRULE are not supported (for example, DTSTART, DTEND, DURATION).
+- Evaluation frequencies must be a day or longer. For shorter evaluation frequencies, use the default monitor schedules.
+
+#### Example: Monitor evaluates on the last day of the month
+```text
+FREQ=MONTHLY;BYMONTHDAY=28,29,30,31;BYSETPOS=-1
+```
+{{< img src="monitors/guide/custom_schedules/RRULE_last_day_month.png" alt="RRULE syntax used in the UI to evaluate the last day of the month" style="width:90%;" >}}
+
+#### Example: Monitor evaluates every other month on the first and last Sunday of the month:
+
+```text
+FREQ=MONTHLY;INTERVAL=2;BYDAY=1SU,-1SU
+```
+
+{{< img src="monitors/guide/custom_schedules/RRULE_month_last_sunday.png" alt="RRULE syntax used in the UI to evaluate every other month on the first and last Sunday of the month" style="width:90%;" >}}
+
+## Alerting behavior of monitors with custom schedule
+
+Monitors using default scheduling run the query with the default evaluation frequency and send alerts based on monitor status transitions (for example, when a monitor goes from WARN to OK or from OK to ALERT).
+
```

Comment:
The timeline below illustrates the behavior of a monitor with default scheduling.  Note that the monitor sends alerts corresponding to status changes.